tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/SDKClasspathContainerInitializer.java

```diff
@@ -72,12 +74,11 @@ public void initialize( IPath containerPath, IJavaProject project ) throws CoreE
             throw new CoreException( ProjectCore.createErrorStatus( msg + SDKClasspathContainer.ID ) );
         }
 
-        PortalBundle bundle = ServerUtil.getPortalBundle(project.getProject());
+        PortalBundle bundle = ServerUtil.getPortalBundle( project.getProject() );
```

Comment:
How does the correct PortalBundle get created if the user fixes the properties?  How does the SDK Classpath COntainer get  re-initialized?   Can you add tests for this?

Reply:
It wounldn't effect on any correct Bundles if the user fixed the wrong.